bibtex-actions-file.el

```diff
@@ -177,11 +180,10 @@ If you use 'org-roam' and 'org-roam-bibtex', you can use
 
 (defun bibtex-actions-file-open-notes-default-org (key entry)
   "Open a note file from KEY and ENTRY."
-  ;; modify when this addressed:
-  ;; https://github.com/org-roam/org-roam-bibtex/issues/211
   (if-let* ((file
              (caar (bibtex-actions-file--files-to-open-or-create
-                    (list key)
+                    key
+                    entry
```

Comment:
I think replacing `entry` with `nil` here should fix the other bug found today, which is due the `file` field which doesn't contain a note getting parsed and then opened. That field is the only reason `entry` is needed by `possible-names`.

Reply:
And the `entry` is only needed there to get the file field. Hmm ...

Reply:
> And the `entry` is only needed there to get the file field. Hmm ...
> 
> Maybe if I remove and inline this I can figure something out.

Inlining will move the same question to `possible-names`. Entry should be passed to `possible-names` if we want to parse the file-field and not otherwise. Setting entry to `nil` is slightly iffy but I think not worth changing `possible-names`.

Reply:
I haven't looked closely yet, but what is entry optional?

Reply:
Optional entry should work but would require changing the order of the arguments.